### ISSUE TITLE
Unable to overwrite module setting in 3.2

### ISSUE THREAD

**Issue by gnomesley** [ivogabe/gulp-typescript#526]
**Expected behavior:**
When I overwrite **module** to "amd" in my config, gulp-typescript should output amd modules. This works in 3.1.7, but is broken in 3.2.

**Actual behavior:**
gulp-typescript emits the default commonjs modules, which is not even in my tsconfig.json.

**Your gulpfile:**

```javascript
var gulp = require("gulp");
var typescript = require("gulp-typescript");

var project = typescript.createProject("tsconfig.json", {module: "amd"});

gulp.task("default", function ()
{
    return gulp.src("src/*.ts")
               .pipe(project())
               .pipe(gulp.dest("dist"));
});
```

**tsconfig.json**

```json
{
  "compilerOptions": {
    "module": "es6",
    "target": "es5",
    "sourceMap": false
  },
  "exclude": [
    "node_modules"
  ]
}
```

I created a minimal repro project here: https://github.com/gnomesley/gulp-typescript-overwrite

**Comment by ivogabe** [comment]
I think that this is fixed with ffd9e689df4572a00aab993552075535455742e3. Can you install gulp-typescript from GitHub and check whether this fixes your problems?

**Comment by gnomesley** [comment]
Seems to be working now. Thanks!

**Comment by ivogabe** [comment]
The fix is included in the new release, 3.2.1.
